modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -734,6 +734,11 @@ public ObjectDefinition getObjectDefinition(long objectDefinitionId)
 		return objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 	}
 
+	@Override
+	public List<ObjectDefinition> getObjectDefinitions(long companyId) {
```

Comment:
Call this `getUnmodifiableSystemObjectDefinitions`. Currently the real behaviour is obscured by the name of the method.